### DISCUSSION TITLE
使用沙盒会导致字体和icon失效，请问有解决方案吗？

### DISCUSSION THREAD

**Discussion by li2446723840** [umijs/qiankun#2970] [Q&A]
使用沙盒会导致字体和icon失效，也会导致子系统使用/deep/的样式失效，请问有什么解决方案。我现在没有使用沙盒，使用的postcss-prefixwrap插件给每个子系统添加class前缀，但是还是有些样式无法兼容，请问有更好的方案吗？

**Comment by L0st1** [comment]
不管是沙箱、严格沙箱、实验性的scoped，实际体验下来，建议不开启沙箱并使用规范来解决，Vue就scoped，React就CSS Modules。
> 像element-plus的话，弹窗组件会默认挂在容器外面，和主应用同一级，就给弹窗给个自定义类型再用`:global()`
加前缀的话，如果没有keep-alive需求可以改主应用的（不用每个子应用都去改自己的）
